### PR TITLE
Added Html and css file for review

### PR DESCRIPTION
**Pull request by collinsmezie**
Js documentation code review by author

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Document</title>
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+  <div class="main-body">
+  <main>
+    <div id="content">
+    <h1 id = "section1">Introduction</h1>
```
Comment (collinsmezie) [inline index.html:40]: Please use space-less equal signs. using space-less equal signs helps to improve code readability.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        <li><a href="#">Declaring Variables</a></li>
+        <li><a href="#">Variable Scope</a></li>
+        <li><a href="#">Global Variables</a></li>
+        <li><a href="#">Constants</a></li>
+        <li><a href="#">Data Types</a></li>
+        <li><a href="#">If...Else Statements</a></li>
+        <li><a href="#">While Statement</a></li>
+        <li><a href="#">Function Declarations</a></li>
+        <li><a href="#">Reference</a></li>
+
```
Comment (collinsmezie) [inline index.html]: Kindly avoid having blank lines and spaces in your code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        <li><a href="#">While Statement</a></li>
+        <li><a href="#">Function Declarations</a></li>
+        <li><a href="#">Reference</a></li>
+        
+     </ul>
+   </nav>
+  </aside>
+</div>
+
+
```
Comment (collinsmezie) [inline index.html:101]: please avoid having blank lines and spaces in your code.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+</body>
+</html>
+
+
+
+
+
+
+
+
```
Comment (collinsmezie) [inline index.html:111]: Please do well to avoid having blank lines and spaces in your code, as this may affect the readability

File: index.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
```
Comment (collinsmezie) [inline index.html:1]: Please do well to avoid having blank lines and spaces in your code, as this may affect the readability

File: index.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
```
Comment (collinsmezie) [inline index.html:12]: please avoid having blank lines and spaces in your code.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+          <li><a href="#">Declaring Variables</a></li>
+          <li><a href="#">Variable Scope</a></li>
+          <li><a href="#">Global Variables</a></li>
+          <li><a href="#">Constants</a></li>
+          <li><a href="#">Data Types</a></li>
+          <li><a href="#">If...Else Statements</a></li>
+          <li><a href="#">While Statement</a></li>
+          <li><a href="#">Function Declarations</a></li>
+          <li><a href="#">Reference</a></li>
+
```
Comment (collinsmezie) [inline index.html:33]: kindly avoid having blank lines and spaces in your code.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+    <p>
+      JavaScript is a very free-form language compared to Java.
+      You do not have to declare all variables, classes, and methods.
+      You do not have to be concerned with whether methods are public, private, or protected, 
+      and you do not have to implement interfaces. Variables, parameters, 
+      and function return types are not explicitly typed.
+    </p>
+    </div>
+  </main>
+
```
Comment (collinsmezie) [inline index.html:97]: please do avoid having blank lines and spaces in your code.

**Review by collinsmezie — COMMENTED** [review]
**### Changes required** :recycle: 
Kindly check the comments below and make the required changes.
**Thanks.**

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @collinsmezie, 

## Congratulations! 🎉 Project Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀

To Highlight:

- [x] The Pull Request has a proper title and description. ✔️
- [x] You have added a descriptive Readme file. Good Job. ✔️
- [x] There is no linter error is present. ✔️
- [x] Project has met all the technical aspects you needed for this project! ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**OPTIONAL**
- [ ] This is just a suggestion so you can apply it if you wish 😁To improve your reviews even further, you could use a couple of emojis and/or gifs in the review. The reason for this is to make the code review more human-friendly 👽🤖

_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**